test/cucumber/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package cucumber provides tools for working with Gherkin.
+// This is a separate package to allow including these tools in both test and test/steps.
+// This package is not named "Gherkin" to avoid collisions with the "gherkin" package from godog.
+package cucumber
```

Comment:
Calling this cucumber makes me think its part of a cucumber library. Thoughts on naming it something like gherkinutil or adding it to helpers?

Reply:
I can't put it in `helpers` or `tests` to avoid cyclic dependencies, so I went with `gherkintools`. I like it, great suggestion!